services/notebook/vql.go

```diff
@@ -0,0 +1,94 @@
+package notebook
+
+import (
+	artifacts_proto "www.velocidex.com/golang/velociraptor/artifacts/proto"
+	"www.velocidex.com/golang/velociraptor/paths"
+	"www.velocidex.com/golang/velociraptor/reporting"
+	"www.velocidex.com/golang/vfilter"
+)
+
+type VqlCell struct {
+}
+
+type VqlProcessor interface {
+	Process(GuiTemplateEngineI, string) (string, error)
+}
+
+type GuiTemplateEngineI interface {
```

Comment:
This is basically the same as https://github.com/Velocidex/velociraptor/blob/23b7def4a330a48846582ac3f1ce97fd81d99a7e/reporting/report.go#L22